### DISCUSSION TITLE
ios移动端网页上，youtube也找不到沉浸式翻译的设置选项

### DISCUSSION THREAD

**Discussion by EvvvaWang** [immersive-translate/immersive-translate#2160] [Help]
### 概述 | Summary

求助，求指引
ios移动端网页上，youtube也找不到沉浸式翻译的设置选项

**Comment by keyboard3** [comment]
你说的字幕的样式设置吗？可以在 https://dash.immersivetranslate.com/#subtitle 这里设置
